conn.go

```diff
@@ -109,6 +109,12 @@ type CloseError struct {
 	Text string
 }
 
+// BufferPool represents a pool of buffers.
```

Comment:
Mention that sync.Pool implements the interface.  Also state that a shared pool must support concurrent Get() and Put().